README.md

```diff
@@ -217,6 +217,8 @@ In order to develop on the Spark Notebook, you'll have to use the `run` command
 
 Use
 ---
+Before the first launch, it may be necessary to add some settings to conf/application.conf. In particular vmArgs can be used to set environment variables for the driver (e.g. `-Dhdp.version=$HDP-Version` if you want to run spark-notebook on a Hortonworks cluster). These are the settings that you would commonly pass via ` spark.driver.extraJavaOptions`.
```

Comment:
maybe bold the reference to hortonworks? So it's clearer that the current example tells more about this use case?